lib/cli/lookup-files.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
+
 /**
  * Contains `lookupFiles`, which takes some globs/dirs/options and returns a list of files.
  * @module
  * @private
  */
 
-var fs = require('node:fs');
-var path = require('node:path');
-var glob = require('glob');
-var errors = require('../errors');
-var createNoFilesMatchPatternError = errors.createNoFilesMatchPatternError;
-var createMissingArgumentError = errors.createMissingArgumentError;
+const fs = require('node:fs');
+const path = require('node:path');
+const glob = require('glob');
+const {
+  createNoFilesMatchPatternError,
+  createMissingArgumentError
+} = require('../errors');
```

Comment:
Can you separate out these unrelated changes, like `var` -> `const`? I'm in favor of them in general but the diff is already pretty big.

If you want to holistically change most/all `var`s to `const`s separately I'd be very in favor.